app/models/trainee.rb

```diff
@@ -5,6 +5,7 @@ class Trainee < ApplicationRecord
   enum record_type: { assessment_only: 0 }
   enum locale_code: { uk: 0, non_uk: 1 }
 
+  has_many :degrees
```

Comment:
i think we'll want to do a `dependent: destroy` here so that the degrees belonging to a trainee are removed if that trainee is removed.

Reply:
Done